database/postgres/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --for=create --timeout=60s
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --all --for=condition=ready --timeout=30s
+
+# Create the kubearchive database
+LOCAL_PORT=5433
+export PGPASSWORD=$(kubectl -n ${NAMESPACE} get secret kubearchive-superuser -o jsonpath='{.data.password}' | base64 --decode)
+kubectl -n ${NAMESPACE} port-forward service/kubearchive-rw ${LOCAL_PORT}:5432 >& /dev/null &
+pid=$!
+
+# Wait for $LOCAL_PORT to become available.
+while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
+    #echo sleeping
+    sleep 0.1
+done
```

Comment:
Since this is an install script, people will likely be watching it to make sure it completes successfully. I would suggest that you add an echo before the wait loop explaining what is happening so they don't have to guess.

Also, if you want to indicate progress, the nerdy way to do that is with a line of dots, created by `echo -n .` inside the loop with a one second wait between attempts. Then a final `echo .` to force a CRLF at the end.

```suggestion
echo Waiting for database port $LOCAL_PORT to become available.
while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
    echo -n .
    sleep 1
done
echo .
```